app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
```

Comment:
so like this https://github.com/ManageIQ/manageiq/pull/15251/commits/7af37090c33b91ea21d0478b8275f00683367a9a ?